### PR TITLE
Pass docker build args for OS and distro

### PR DESCRIPTION
**Pull request by manju956**
Signed-off-by: manju956 <manjunath.a.c@intel.com>

### CODE REVIEW

File: docker/compose/avalon-pool.yaml [hunk 1]
```diff
@@ -74,6 +76,8 @@ services:
         - https_proxy
         - no_proxy
         - ENCLAVE_TYPE=wpe
+        - IMAGE=$IMAGE
```
Comment (rranjan3) [inline docker/compose/avalon-pool.yaml:79]: Would not be required. Will be taken up from default docker-compose.yaml.

File: docker/compose/avalon-multi-pool.yaml [hunk 2]
```diff
@@ -79,6 +81,8 @@ services:
         - https_proxy
         - no_proxy
         - ENCLAVE_TYPE=wpe
+        - IMAGE=$IMAGE
```
Comment (rranjan3) [inline docker/compose/avalon-multi-pool.yaml:84]: Would rely on defaults from docker-compose.yaml. Double check for all other attributes as well.

**Review by rranjan3 — COMMENTED** [review]
Minor changes required.

File: docker/compose/avalon-pool.yaml (hunk 1 above)
Comment (manju956) [reply docker/compose/avalon-pool.yaml]: Yes. Removed it along with other duplicate attriutes.

File: docker/compose/avalon-multi-pool.yaml (hunk 2 above)
Comment (manju956) [reply docker/compose/avalon-multi-pool.yaml]: Removed all the duplicate attributes for the overriden service.

**Review by rranjan3 — APPROVED** [review]

**Review by pankajgoyal2 — APPROVED** [review]

**Pull request merged**